packages/shopify-codemod/transforms/split-if-statement-assignment-parameter.js

```diff
@@ -0,0 +1,30 @@
+export default function splitIfStatementAssignmentParameter({source}, {jscodeshift: j}, {printOptions = {}}) {
+  const sourceAST = j(source);
+  sourceAST
+    .find(j.IfStatement, {
+      type: j.IfStatement.name,
+      test: {
+        type: j.AssignmentExpression.name,
+      },
+    }).forEach((nodePath) => {
+      const args = nodePath.get('test').node;
```

Comment:
You `get` the test node again in a few lines, maybe store it as a temporary variable?